lib/sisimai/bite/email/sendmail.rb

```diff
@@ -0,0 +1,290 @@
+module Sisimai::Bite::Email
+  module Sendmail
+    # Sisimai::Bite::Email::Sendmail parses a bounce email which created by
+    # v8 Sendmail. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Sendmail.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/\AMail Delivery Subsystem/,
+        :subject => %r/(?:see transcript for details\z|\AWarning: )/,
+      }
+      # Error text regular expressions which defined in sendmail/savemail.c
+      #   savemail.c:1040|if (printheader && !putline("   ----- Transcript of session follows -----\n",
+      #   savemail.c:1041|          mci))
+      #   savemail.c:1042|  goto writeerr;
+      #
+      Re1 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->